services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/domain/PublicHanke.kt

```diff
@@ -101,8 +101,8 @@ fun hankeToPublic(hanke: Hanke): PublicHanke {
         hanke.hankeTunnus!!,
         hanke.nimi!!,
         hanke.kuvaus!!,
-        hanke.alkuPvm!!,
-        hanke.loppuPvm!!,
+        hanke.alkuPvm,
+        hanke.loppuPvm,
```

Comment:
A public hanke should always have a start and end date, doesn't it?

Reply:
Yes.